pkg/oc/clusterup/run_self_hosted.go

```diff
@@ -181,6 +185,15 @@ func (c *ClusterUpConfig) StartSelfHosted(out io.Writer) error {
 		return err
 	}
 
+	// bootstrap the service-ca operator with the legacy service-signer.crt and key.
+	err = createServiceCASigningSecret(clientConfig,
```

Comment:
@mfojtik I think we tolerate this for 3.11 and remove it after master re-opens.  The webconsole won't be present and the router won't either.